### PR TITLE
An-5233-gold-feeback-updates

### PR DESCRIPTION
**Pull request by eric-laurello**
- models/gold/core/core__fact_transfers.sql
  -change to a table, remove the 0 amount rows
- models/gold/defi/defi__ez_bridge_activity.sql
  - new model, add decimals and prices
- models/gold/defi/defi__ez_dex_swaps.sql
  - fix join to prices
- models/gold/defi/defi__fact_bridge_activity.sql
  - update to a table
- models/silver/core/silver__transactions.sql
  - performance improvements
- models/silver/core/silver__transfers.sql
  - remove SO
- models/silver/defi/bridge/silver__bridge*
  - create a combined view, add event index
- models/silver/defi/dex/silver__dex_swaps*
  - performance upgates
 
`dbt run -m  models/gold/defi/defi__ez_bridge_activity.sql   models/gold/defi/defi__fact_bridge_activity.sql models/silver/defi/bridge/silver__bridge_celer_transfers.sql models/silver/defi/bridge/silver__bridge_combined.sql models/silver/defi/bridge/silver__bridge_layerzero_transfers.sql models/silver/defi/bridge/silver__bridge_mover_transfers.sql  --full-refresh `
 
 `dbt run -m models/gold/core/core__fact_transfers.sql models/gold/defi/defi__ez_dex_swaps.sql models/silver/core/silver__transactions.sql models/silver/core/silver__transfers.sql models/silver/defi/dex/silver__dex_swaps_animeswap.sql models/silver/defi/dex/silver__dex_swaps_auxexchange.sql models/silver/defi/dex/silver__dex_swaps_cellana.sql models/silver/defi/dex/silver__dex_swaps_cetus.sql models/silver/defi/dex/silver__dex_swaps_hippo.sqlmodels/silver/defi/dex/silver__dex_swaps_liquidswap.sql models/silver/defi/dex/silver__dex_swaps_pancake.sql models/silver/defi/dex/silver__dex_swaps_sushi.sql
 models/silver/defi/dex/silver__dex_swaps_thala.sql`

### CODE REVIEW

File: models/gold/defi/defi__ez_bridge_activity.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+version: 2
```
Comment (tarikceric) [inline models/gold/defi/defi__ez_bridge_activity.yml:1]: getting a few of the same errors across tables from `dbt test`, ie
```
Compilation Error in test block_number_not_null (models/gold/defi/defi__ez_bridge_activity.yml)
  macro 'dbt_macro__test_not_null' takes no keyword argument 'Where'
``` 
^ should be lowercase
```
Database Error in test bridge_address_not_null (models/gold/defi/defi__ez_bridge_activity.yml)
  001003 (42000): SQL compilation error:
  syntax error line 1 at position 67 unexpected '.'.
```

File: models/gold/core/core__fact_transfers.sql [hunk 2]
```diff
@@ -1,6 +1,11 @@
 {{ config(
-    materialized = 'view',
-    tags = ['core']
+    materialized = 'incremental',
+    unique_key = ['tx_hash','event_index','block_timestamp::DATE'],
```
[the remark this answers is not in the document]
Comment (eric-laurello) [reply models/gold/core/core__fact_transfers.sql:3]: doing it this way allows for the merge to use the block_timestamp::Date cluster key

File: models/gold/defi/defi__ez_bridge_activity.yml (hunk 1 above)
Comment (eric-laurello) [reply models/gold/defi/defi__ez_bridge_activity.yml:1]: weird I thought i had this working. Updated casing and test names

File: models/silver/defi/bridge/silver__bridge_combined.sql [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+FROM
+    ({% for models in models %}
+    SELECT
+     *
+    FROM
+        {{ models [1] }}
+
+        {% if not loop.last %}
+
+{% if is_incremental() %}
```
[the remark this answers is not in the document]
Comment (eric-laurello) [reply models/silver/defi/bridge/silver__bridge_combined.sql:46]: good catch, needs to be removed - it technically does nothing, just gets skipped

**Review by tarikceric — APPROVED** [review]

**Pull request merged**